src/pubsub/ua_pubsub_keystorage.c

```diff
@@ -75,6 +78,13 @@ UA_PubSubKeyStorage_delete(UA_Server *server, UA_PubSubKeyStorage *keyStorage) {
     if(keyStorage->securityGroupID.data)
         UA_String_clear(&keyStorage->securityGroupID);
 
+    /* delete SKS client config from keystorage */
+    if(keyStorage->sksConfig.sksClientConfig) {
+        UA_ClientConfig_delete(keyStorage->sksConfig.sksClientConfig);
+    }
+
+    memset(keyStorage, 0, sizeof(UA_PubSubKeyStorage));
```

Comment:
No need to memset zero if the memory is freed right afterwards.
The paranoid can configure their malloc / free to overwrite memory once freed in the background.